Assets/Scripts/Player/PlayerLook.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerLook : MonoBehaviour
+{
+    [SerializeField] private Transform m_camera;
+    [SerializeField] private Transform m_player;
+    [Space]
+    [SerializeField] private float m_mouseSensitivity = 50f;
+
+    public bool m_hideLockCamera = true;
+
+    private float m_xRotation;
+
+    private void Start()
+    {
+        CursorCheck();
+    }
+
+    private void FixedUpdate()
+    {
+        MouseControl();
```

Comment:
Rename MouseControl to something else, a method does something, MouseControl does not tell me what it does.